index.d.ts

```diff
@@ -120,6 +120,8 @@ declare class Agent implements Taggable, StartSpanFn {
   registerMetric(name: string, labels: Labels, callback: Function): void;
```

Comment:
I see here we've created a new type, `Outcome`, that can be one of three string values.  We've also used the type to add a new `outcome` property to the `GenericSpan` class.